storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -196,6 +198,8 @@ private Object loadWorker(IStateStorage stateStorage, IStormClusterState stormCl
         throws Exception {
         workerState = new WorkerState(conf, context, topologyId, assignmentId, supervisorIfaceSupplier, port, workerId,
                                       topologyConf, stateStorage, stormClusterState, autoCreds, metricRegistry);
+        this.heatbeatMeter = metricRegistry.meter("doHeartbeat-calls", workerState.getWorkerTopologyContext(),
+                "__system", -1, "default");
```

Comment:
It is better to not hardcode it. We can use `SYSTEM_COMPONENT_ID`, `SYSTEM_TASK_ID`. And it looks like it doesn't make much sense to have `streamId` here.

Reply:
Would you prefer I pass in "" for streamId?  Or are you indicating we should have a metric name convention that allows skipping inclusion of stream?

Reply:
I see there is already a metricName function https://github.com/apache/storm/blob/master/storm-client/src/jvm/org/apache/storm/metrics2/StormMetricRegistry.java#L244 which doesn't take streamId

Reply:
Also I feel like the metrics should have the key before the value, for example, we now have
`xx.<compId>.<taskId>.yy`. 
I think it should be `xx.componentId.<compId>.taskId.<taskId>.yy` so the values are more readable. But this is a separate issue.

Reply:
@Ethanlm - updated